app/retail/templates/shared/nav.html

```diff
@@ -87,18 +87,10 @@
       {% else %}
       <a class="nav-link{% if active == 'tools' %} selected{% endif %} interior" href="{% url "tools" %}">{% trans "Tools" %}</a>
       {% endif %}
-      {% if request.path != '/' %}
-        <a href="{% url "new_funding" %}" class="nav-link interior button--fund">
-          <i class="fab fa-ethereum mr-1" role="aria-hidden"></i>
-          {% trans "FUND ISSUE" %}
-        </a>
-      {% else %}
-        <a href="{% url "new_funding" %}" class="btn btn-gc-blue interior button--fund">
-          <i class="fab fa-ethereum mr-1" role="aria-hidden"></i>
-          {% trans "FUND ISSUE" %}
-        </a>
-      {% endif %}
```

Comment:
Could you update it to look like `{% if request.path != '/' %} nav-link {% else %} btn btn-gc-blue{% endif %}` ?
We need the diff classes. 

This change in this PR prevent the button styling from turning up on scroll on index

![untitled](https://user-images.githubusercontent.com/5358146/49643029-a8b0b400-fa3a-11e8-8ab9-c85c2374d115.gif)

Reply:
Ah my bad, didn't notice the transition. Updating

I think adding `btn` class overwrites it's behavior and therefore it's centered. Finding a workaround now! 😄